go-controller/pkg/ovn/base_network_controller_policy.go

```diff
@@ -1393,6 +1408,14 @@ func (bnc *BaseNetworkController) handlePeerNamespaceSelectorAdd(np *networkPoli
 	var errors []error
 	for _, obj := range objs {
 		namespace := obj.(*kapi.Namespace)
+		netinfo, err := bnc.networkManager.GetActiveNetworkForNamespace(namespace.Name)
+		if err != nil {
+			klog.Warningf("Could not get active network for namespace %s: %v", namespace.Name, err)
+			continue
+		}
+		if bnc.GetNetworkName() != netinfo.GetNetworkName() {
+			continue
+		}
```

Comment:
this check must be removed as it's breaking mutlinet policies on secondary networks.